admin.py

```
@@ -2,19 +2,38 @@
+    if not expiry:
+        return False
+    if datetime.now(timezone.utc) > expiry:
+        del active_tokens[token]
+        return False
+    return True
 
 @router.post("/admin", response_class=HTMLResponse)
 async def admin_login_post(request: Request, password: str = Form(...)):
     if bcrypt.checkpw(password.encode('utf-8'), pw):
-        token = 'secure'
+        token = secrets.token_urlsafe(128);
```

Ruff (0.12.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)